app/uk/gov/hmrc/pillar2/controllers/RegistrationCacheController.scala

```diff
@@ -68,11 +66,9 @@ class RegistrationCacheController @Inject() (
   }
 
   def lastUpdated(id: String): Action[AnyContent] = authenticate.async { implicit request =>
-    implicit val hc: HeaderCarrier = HeaderCarrierConverter.fromRequestAndSession(request, request.session)
-
-    logger.debug(s"[Session ID: ${SessionIdHelper.sessionId(hc)}] - controllers.RegistrationCacheController.lastUpdated: Authorised Request " + id)
+    logger.debug(s"- controllers.RegistrationCacheController.lastUpdated: Authorised Request " + id)
```

Comment:
Can remove string interpolation.

Reply:
Removed the whole logger because its all useless really